Doc/c-api/structures.rst

```diff
@@ -482,6 +519,52 @@ definition with the same method name.
    Equivalent to ``PyCMethod_New(ml, self, NULL, NULL)``.
 
 
+.. c:function:: int PyCFunction_GetFlags(PyObject *func)
+
+   Get the function flags on *func* as they were passed to
+   :c:member:`~PyMethodDef.ml_flags`.
+
+   This function returns the function's flags on success, and ``-1`` with an
+   exception set on failure.
+
+
+.. c:function:: int PyCFunction_GET_FLAGS(PyObject *func)
+
+   This is the same as :c:func:`PyCFunction_GetFlags`, but without error
+   checking.
+
+
+.. c:function:: PyCFunction PyCFunction_GetFunction(PyObject *func)
+
+   Get the function pointer on *func* as it was passed to
```

Comment:
Please indicate whether passing a non-function object sets an exception or causes a crash (to make the distinction with `PyCFunction_GET_FUNCTION`)